document/js-api/index.bs

```diff
@@ -1764,8 +1764,10 @@ In practice, an implementation may run out of resources for valid modules below
 <li>The maximum number of data segments defined in a module is 100,000.</li>
 
 <li>The maximum number of tables, including declared or imported tables, is 100,000.</li>
-<li>The maximum size of a table is 10,000,000.</li>
+<li>The maximum initial size of a table is 10,000,000.</li>
 <li>The maximum number of table entries in any table initialization is 10,000,000.</li>
+<li>The maximum initial size of a 32-bit memory is 65,536 pages (4 GiB).</li>
```

Comment:
It is impossible to request a 32-bit memory larger than 4GB, so this limit doesn't really mean anything. It should probably be removed.